src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -700,6 +700,15 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
         GPU_DEBUG_IF(debug_config->verbose >= 1) {
             GPU_DEBUG_COUT << "Execute " << inst->id() << ", memory type: "
                            << inst->output_memory().get_allocation_type() << std::endl;
+            auto debug_in_out_mem_addr = [&]() {
+                for (size_t i = 0; i < get_primitive(inst->id())->dependencies().size(); i++) {
+                    auto& in_mem = get_primitive(inst->id())->dep_memory(i);
+                    GPU_DEBUG_COUT << "In " << in_mem.buffer_ptr() << std::endl;
+                }
+                auto& out_mem = get_primitive(inst->id())->output_memory();
+                GPU_DEBUG_COUT << "Out " << out_mem.buffer_ptr() << std::endl;
+            };
+            debug_in_out_mem_addr();
```

Comment:
why did you define a lambda here? Could you print those buf addr in a single line?

Reply:
updated it